test/e2e/performanceprofile/functests/utils/deployments/deployments.go

```diff
@@ -0,0 +1,63 @@
+package deployments
+
+import (
+	"context"
+	"time"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewTestDeployment(name, namespace string, podTemplate *corev1.Pod, opts ...func(dp *appsv1.Deployment)) *appsv1.Deployment {
```

Comment:
lets follow the convention from before and call it `Make` so from outside it called:
`deployments.Make(With...)`

Reply:
Done